src/style.css

```
@@ -1 +1,49 @@
 /* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.main {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.stars {
+  display: flex;
+  justify-content: center;
+}
+
+.stars__star {
+  background-image: url('images/star.svg');
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star:nth-child(-n + 0) {
+  background-image: url('images/star-active.svg');
+}
+
+.stars--1 .stars__star:nth-child(-n + 1) {
+  background-image: url('images/star-active.svg');
+}
+
+.stars--2 .stars__star:nth-child(-n + 2) {
+  background-image: url('images/star-active.svg');
+}
+
+.stars--3 .stars__star:nth-child(-n + 3) {
+  background-image: url('images/star-active.svg');
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url('images/star-active.svg');
+}
+
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url('images/star-active.svg');
+}
```

The selectors can be written in a more compact form:

```suggestion
.stars--0 .stars__star:nth-child(-n + 0),
.stars--0 .stars__star:nth-child(-n + 1),
.stars--0 .stars__star:nth-child(-n + 2),
.stars--0 .stars__star:nth-child(-n + 3),
.stars--0 .stars__star:nth-child(-n + 4),
.stars--0 .stars__star:nth-child(-n + 5) {
background-image: url('images/star-active.svg');
}

```